desktop/package/package.gradle

```diff
@@ -366,6 +366,57 @@ task packageInstallers {
                         " -vvvv" +
                         " --deep" +
                         " '${binariesFolderPath}/Bisq-${appVersion}.dmg'")
+
+                // macOS step 6: Upload for notarization
+                // See https://developer.apple.com/documentation/xcode/notarizing_macos_software_before_distribution/customizing_the_notarization_workflow#3087734
+                String envVariableAcUsername = "$System.env.BISQ_PACKAGE_NOTARIZATION_AC_USERNAME"
+                String envVariableAscProvider = "$System.env.BISQ_PACKAGE_NOTARIZATION_ASC_PROVIDER"
+                def uploadForNotarizationOutput = executeCmd("xcrun altool --notarize-app" +
+                        " --primary-bundle-id 'network.bisq.CAT'" +
```

Comment:
We should envize the primary bundle id argument as well, as it has to be unique for each developer (CAT -> Christoph ATteneder) who wants to publish a binary.